src/Aquifer.API/Endpoints/Projects/Get/Endpoint.cs

```diff
@@ -33,6 +33,10 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
         var items = await GetProjectItemsAsync(req.ProjectId, ct);
 
         project.Items = items.OrderBy(x => x.SortOrder).ThenBy(x => x.EnglishLabel);
+
+        var projectResourceItems = project.Items as ProjectResourceItem[] ?? project.Items;
+        project.Counts = new ProjectResourceStatusCounts(projectResourceItems.Select(pri => (pri.Status, pri.WordCount)).ToList());
```

Comment:
This looks like it can be simplified.
```suggestion
        project.Counts = new ProjectResourceStatusCounts(project.Items.Select(pri => (pri.Status, pri.WordCount)).ToList());
```